### PR TITLE
new LOAD/VLOAD/BLOAD etc documentation (W.I.P.)

### PR DESCRIPTION
**Pull request by irmen**
As I understand it, LOAD cannot load into Vera RAM, there is VLOAD to do that.

So the additional functionality to load data smartly into HIRAM is now documented instead. I based this change on what was written on the forum here https://www.commanderx16.com/forum/index.php?/topic/2064-r39-patched-kernal-to-fix-load-into-hiram-functionality/#comment-16153  where the fix for this was discussed.

### CODE REVIEW

**Comment by irmen** [comment]
@ZeroByteOrg could you perhaps review this?

File: X16 Reference - 03 - BASIC.md [hunk 1]
```diff
@@ -419,18 +419,18 @@ The numeric constants parser supports both hex (`$`) and binary (`%`) literals,
 
 The size of hex and binary values is only restricted by the range that can be represented by BASIC's internal floating point representation.
 
-#### LOAD into VRAM
+#### LOAD into HIRAM
```
Comment (ZeroByteOrg) [inline X16 Reference - 03 - BASIC.md:422]: I think it makes sense to keep this section focused on loading directly to VRAM as this is indeed a new concept for X16 vs anyone with CBM experience....

Comment (irmen) [reply X16 Reference - 03 - BASIC.md:422]: Enhanced LOAD functionality:
1) user-specified LOAD address: (LOAD supports syntax LOAD "myfile.prg",device,bank,address)
 - ignores the 2-byte PRG header address and loads the file into memory at the address specified in the final argument. If the address is in the HiRam bank window, LOAD will automatically extend the data into the next banks as necessary.
Headerless (binary) LOAD:
BLOAD and BVLOAD are available as "binary load" equivalents for loading files which do not have the 2-byte PRG header. These commands will not skip the first two bytes when loading. 
Right after VRAM loading.

Comment (irmen) [reply X16 Reference - 03 - BASIC.md:422]: see also https://github.com/commanderx16/x16-rom/pull/312

**Pull request closed without merging**